api/server/router/network/network_routes.go

```diff
@@ -17,6 +17,68 @@ import (
 	"github.com/docker/libnetwork"
 )
 
+// supportedFilters predefined some supported filter handler function
+type filterHandler func(libnetwork.Network, []string) (bool, error)
```

Comment:
can we put all this filtering logic in a separated file, like `api/server/router/network/filters.go`? just to keep them separated from the routes :smile: